### PR TITLE
Workflows / CI: updated to Rails 7

### PR DESCRIPTION
**Pull request by Alexander-Senko**
Updated _dummy_ application to Rails 7.2.
Loading process has changed in Rails 7.

## To-Dos

- [x] tests

## References

* Depends on #935

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,32 +17,30 @@ platforms :jruby do
...
-if RUBY_VERSION >= "2.7.0"
-  gem "mongoid", github: "mongodb/mongoid"
-elsif RUBY_VERSION >= "2.6.0"
-  gem "mongoid", "~> 8.1"
-else
-  gem "mongoid", "~> 7.2"
-end
+gem 'mongoid' unless
+    rails_version == 'edge'
+gem 'active_model_serializers'
```
Comment (Alexander-Senko) [inline Gemfile:31]: Moved here to be loaded on application start.

File: spec/dummy/app/decorators/mongoid_post_decorator.rb [hunk 2]
```diff
@@ -1,4 +1,2 @@
-if defined?(Mongoid)
-  class MongoidPostDecorator < Draper::Decorator
-  end
+class MongoidPostDecorator < Draper::Decorator
```
Comment (Alexander-Senko) [inline spec/dummy/app/decorators/mongoid_post_decorator.rb:1]: The file should define the class when the app is _eager loaded_.

**Review by y-yagi — APPROVED** [review]
LGTM!

**Comment by Alexander-Senko** [comment]
@y-yagi, I'm sorry, but I had to re-push after getting conflicts (despite they have been auto-resolved).

**Review by y-yagi — APPROVED** [review]
👍

**Pull request merged**